lib/index.js

```diff
@@ -5,8 +5,22 @@ const server = require('http').Server
 const getRawBody = require('raw-body')
 const typer = require('media-typer')
 const isStream = require('isstream')
+const chalk = require('chalk')
+const jsome = require('jsome')
 
 const DEV = 'development' === process.env.NODE_ENV
```

Comment:
It'd be cool to document in the README that we do the nicer logging upon _explicit_ `NODE_ENV = development` setting